--- ./config/index.php~	2005-05-07 00:41:01.325841276 +0300
+++ ./config/index.php	2005-05-07 00:41:03.208531361 +0300
@@ -427,7 +427,7 @@
 		$wgCommandLineMode = false;
 		chdir( ".." );
 		eval($local);
-		$wgDBadminuser = "root";
+		$wgDBadminuser = "mysql";
 		$wgDBadminpassword = $conf->RootPW;
 		$wgDBprefix = $conf->DBprefix;
 		$wgCommandLineMode = true;
@@ -438,7 +438,7 @@
 		require_once( "maintenance/InitialiseMessages.inc" );
 
 		$wgTitle = Title::newFromText( "Installation script" );
-		$wgDatabase = Database::newFromParams( $wgDBserver, "root", $conf->RootPW, "", 1 );
+		$wgDatabase = Database::newFromParams( $wgDBserver, $wgDBadminuser, $conf->RootPW, "", 1 );
 		$wgDatabase->ignoreErrors(true);
 
 		@$myver = mysql_get_server_info( $wgDatabase->mConn );
