diff -urN mediawiki-1.18.0.orig/includes/installer/Installer.php mediawiki-1.18.0/includes/installer/Installer.php
--- mediawiki-1.18.0.orig/includes/installer/Installer.php	2011-11-18 01:42:24.000000000 +0100
+++ mediawiki-1.18.0/includes/installer/Installer.php	2011-12-28 20:04:35.000000000 +0100
@@ -464,7 +464,7 @@
 		global $IP;
 
 		wfSuppressWarnings();
-		$_lsExists = file_exists( "$IP/LocalSettings.php" );
+		$_lsExists = file_exists( "/etc/webapps/mediawiki/LocalSettings.php" );
 		wfRestoreWarnings();
 
 		if( !$_lsExists ) {
@@ -473,7 +473,7 @@
 		unset($_lsExists);
 
 		require( "$IP/includes/DefaultSettings.php" );
-		require( "$IP/LocalSettings.php" );
+		require( "/etc/webapps/mediawiki/LocalSettings.php" );
 		if ( file_exists( "$IP/AdminSettings.php" ) ) {
 			require( "$IP/AdminSettings.php" );
 		}
diff -urN mediawiki-1.18.0.orig/includes/Setup.php mediawiki-1.18.0/includes/Setup.php
--- mediawiki-1.18.0.orig/includes/Setup.php	2011-10-26 20:47:09.000000000 +0200
+++ mediawiki-1.18.0/includes/Setup.php	2011-12-28 20:04:34.000000000 +0100
@@ -308,7 +308,7 @@
 }
 
 if ( $wgInvalidateCacheOnLocalSettingsChange ) {
-	$wgCacheEpoch = max( $wgCacheEpoch, gmdate( 'YmdHis', @filemtime( "$IP/LocalSettings.php" ) ) );
+	$wgCacheEpoch = max( $wgCacheEpoch, gmdate( 'YmdHis', @filemtime( "/etc/webapps/mediawiki/LocalSettings.php" ) ) );
 }
 
 if ( $wgAjaxUploadDestCheck ) {
diff -urN mediawiki-1.18.0.orig/maintenance/checkSyntax.php mediawiki-1.18.0/maintenance/checkSyntax.php
--- mediawiki-1.18.0.orig/maintenance/checkSyntax.php	2011-07-06 23:57:44.000000000 +0200
+++ mediawiki-1.18.0/maintenance/checkSyntax.php	2011-12-28 20:04:37.000000000 +0100
@@ -153,8 +153,8 @@
 		}
 
 		// Manually add two user-editable files that are usually sources of problems
-		if ( file_exists( "$IP/LocalSettings.php" ) ) {
-			$this->mFiles[] = "$IP/LocalSettings.php";
+		if ( file_exists( "/etc/webapps/mediawiki/LocalSettings.php" ) ) {
+			$this->mFiles[] = "/etc/webapps/mediawiki/LocalSettings.php";
 		}
 		if ( file_exists( "$IP/AdminSettings.php" ) ) {
 			$this->mFiles[] = "$IP/AdminSettings.php";
diff -urN mediawiki-1.18.0.orig/maintenance/Maintenance.php mediawiki-1.18.0/maintenance/Maintenance.php
--- mediawiki-1.18.0.orig/maintenance/Maintenance.php	2011-11-15 18:29:10.000000000 +0100
+++ mediawiki-1.18.0/maintenance/Maintenance.php	2011-12-28 20:04:37.000000000 +0100
@@ -961,7 +961,7 @@
 		} elseif ( defined("MW_CONFIG_FILE") ) {
 			$settingsFile = MW_CONFIG_FILE;
 		} else {
-			$settingsFile = "$IP/LocalSettings.php";
+			$settingsFile = "/etc/webapps/mediawiki/LocalSettings.php";
 		}
 		if ( isset( $this->mOptions['wiki'] ) ) {
 			$bits = explode( '-', $this->mOptions['wiki'] );
diff -urN mediawiki-1.18.0.orig/includes/WebStart.php mediawiki-1.18.0/includes/WebStart.php
--- mediawiki-1.18.0.orig/includes/WebStart.php	2011-06-28 20:21:59.000000000 +0200
+++ mediawiki-1.18.0/includes/WebStart.php	2011-12-28 20:31:51.000000000 +0100
@@ -119,7 +119,7 @@
 	MWFunction::call( MW_CONFIG_CALLBACK );
 } else {
 	if ( !defined( 'MW_CONFIG_FILE' ) ) {
-		define('MW_CONFIG_FILE', MWInit::interpretedPath( 'LocalSettings.php' ) );
+		define('MW_CONFIG_FILE', '/etc/webapps/mediawiki/LocalSettings.php');
 	}
 
 	# LocalSettings.php is the per site customization file. If it does not exist
