--- mediawiki-1.6.3/config/index.php~	2006-04-27 20:38:23.367393982 +0300
+++ mediawiki-1.6.3/config/index.php	2006-04-27 20:39:12.668492466 +0300
@@ -494,7 +494,7 @@
 		$dbclass = 'Database'.ucfirst($conf->DBtype);
 		require_once("$dbclass.php");
 		$wgDBtype = $conf->DBtype;
-		$wgDBadminuser = "root";
+		$wgDBadminuser = "mysql";
 		$wgDBadminpassword = $conf->RootPW;
 		$wgDBprefix = $conf->DBprefix;
 		$wgCommandLineMode = true;
--- ./config/index.php~	2006-02-03 19:26:11.000000000 +0200
+++ ./config/index.php	2006-02-03 19:36:19.036054719 +0200
@@ -383,7 +383,7 @@
 	$conf->DBpassword2 = importPost( "DBpassword2" );
 	$conf->DBprefix = importPost( "DBprefix" );
 	$conf->DBmysql5 = (importPost( "DBmysql5" ) == "true") ? "true" : "false";
-	$conf->RootUser = importPost( "RootUser", "root" );
+	$conf->RootUser = importPost( "RootUser", "mysql" );
 	$conf->RootPW = importPost( "RootPW", "-" );
 	$conf->LanguageCode = importPost( "LanguageCode", "en" );
 	$conf->SysopName = importPost( "SysopName", "WikiSysop" );
