--- mediawiki-1.9.0/config/index.php~	2007-01-10 23:35:22.257272505 +0200
+++ mediawiki-1.9.0/config/index.php	2007-01-10 23:36:41.239037706 +0200
@@ -503,7 +503,7 @@
 	$conf->SysopName = importPost( "SysopName", "WikiSysop" );
 	$conf->SysopPass = importPost( "SysopPass" );
 	$conf->SysopPass2 = importPost( "SysopPass2" );
-	$conf->RootUser = importPost( "RootUser", "root" );
+	$conf->RootUser = importPost( "RootUser", "mysql" );
 	$conf->RootPW = importPost( "RootPW", "" );
 	$useRoot = importCheck( 'useroot', false );
 
