--- mediawiki-1.6.3/config/index.php~	2006-04-27 20:38:23.367393982 +0300
+++ mediawiki-1.6.3/config/index.php	2006-04-27 20:39:12.668492466 +0300
@@ -494,7 +494,7 @@
 		$dbclass = 'Database'.ucfirst($conf->DBtype);
 		require_once("$dbclass.php");
 		$wgDBtype = $conf->DBtype;
-		$wgDBadminuser = "root";
+		$wgDBadminuser = "mysql";
 		$wgDBadminpassword = $conf->RootPW;
 		$wgDBprefix = $conf->DBprefix;
 		$wgCommandLineMode = true;
--- mediawiki-1.7.0/config/index.php~	2006-07-08 14:05:35.057049314 +0300
+++ mediawiki-1.7.0/config/index.php	2006-07-08 14:06:04.637710603 +0300
@@ -486,7 +486,7 @@
 	## MySQL specific:
 	$conf->DBprefix     =  importPost( "DBprefix" );
 	$conf->DBmysql5     = (importPost( "DBmysql5" ) == "true") ? "true" : "false";
-	$conf->RootUser     =  importPost( "RootUser", "root" );
+	$conf->RootUser     =  importPost( "RootUser", "mysql" );
 	$conf->RootPW       =  importPost( "RootPW", "-" );
 	$conf->LanguageCode =  importPost( "LanguageCode", "en" );
 
