--- ./config/index.php~	2005-05-07 00:41:01.325841276 +0300
+++ ./config/index.php	2005-05-07 00:41:03.208531361 +0300
@@ -427,7 +427,7 @@
 		$wgCommandLineMode = false;
 		chdir( ".." );
 		eval($local);
-		$wgDBadminuser = "root";
+		$wgDBadminuser = "mysql";
 		$wgDBadminpassword = $conf->RootPW;
 		$wgDBprefix = $conf->DBprefix;
 		$wgCommandLineMode = true;
--- ./config/index.php~	2006-02-03 19:26:11.000000000 +0200
+++ ./config/index.php	2006-02-03 19:36:19.036054719 +0200
@@ -383,7 +383,7 @@
 	$conf->DBpassword2 = importPost( "DBpassword2" );
 	$conf->DBprefix = importPost( "DBprefix" );
 	$conf->DBmysql5 = (importPost( "DBmysql5" ) == "true") ? "true" : "false";
-	$conf->RootUser = importPost( "RootUser", "root" );
+	$conf->RootUser = importPost( "RootUser", "mysql" );
 	$conf->RootPW = importPost( "RootPW", "-" );
 	$conf->LanguageCode = importPost( "LanguageCode", "en" );
 	$conf->SysopName = importPost( "SysopName", "WikiSysop" );
