--- mediawiki-1.12.0/config/index.php	2008-03-27 14:17:49.314569727 +0200
+++ mediawiki-1.12.0/config/index.php	2008-03-27 14:18:45.385792838 +0200
@@ -196,7 +196,7 @@
 
 /* Check for existing configurations and bug out! */
 
-if( file_exists( "../LocalSettings.php" ) ) {
+if( file_exists( "" ) ) {
 	$script = defined('MW_INSTALL_PHP5_EXT') ? 'index.php5' : 'index.php';
  dieout( "<p><strong>Setup has completed, <a href='../$script'>your wiki</a> is configured.</strong></p>
 
@@ -214,7 +214,7 @@
 
 	<p>In order to configure the wiki you have to make the <tt>config</tt> subdirectory
 	writable by the web server. Once configuration is done you'll move the created
-	<tt>LocalSettings.php</tt> to the parent directory, and for added safety you can
+	<tt>LocalSettings.php</tt> to the /etc/webapps/mediawiki directory, and for added safety you can
 	then remove the <tt>config</tt> subdirectory entirely.</p>
 
 	<p>To make the directory writable on a Unix/Linux system:</p>
@@ -1471,7 +1471,7 @@
 <div class="success-box">
 <p>
 <span class="success-message">Installation successful!</span>
-Move the <tt>config/LocalSettings.php</tt> file to the parent directory, then follow
+Move the <tt>/etc/webapps/mediawiki/LocalSettings.php</tt> file to the parent directory, then follow
 <a href="../$script"> this link</a> to your wiki.</p>
 <p>You should change file permissions for <tt>LocalSettings.php</tt> as required to
 prevent other users on the server reading passwords and altering configuration data.</p>
--- mediawiki-1.8.2/includes/WebStart.php~	2006-10-14 03:06:33.000000000 +0300
+++ mediawiki-1.8.2/includes/WebStart.php	2006-10-16 00:49:09.671858850 +0300
@@ -65,7 +65,7 @@
 # LocalSettings.php is the per site customization file. If it does not exit
 # the wiki installer need to be launched or the generated file moved from
 # ./config/ to ./
-if( !file_exists( './LocalSettings.php' ) ) {
+if( !file_exists( '/etc/webapps/mediawiki/LocalSettings.php' ) ) {
 	$IP = '.';
 	require_once( './includes/DefaultSettings.php' ); # used for printing the version
 	require_once( './includes/templates/NoLocalSettings.php' );
@@ -73,7 +73,7 @@
 }
 
 # Include this site setttings
-require_once( './LocalSettings.php' );
+require_once( '/etc/webapps/mediawiki/LocalSettings.php' );
 wfProfileOut( 'WebStart.php-conf' );
 
 if ( !defined( 'MW_NO_SETUP' ) ) {
--- mediawiki-1.8.2/maintenance/archives/upgradeWatchlist.php~	2006-10-14 03:05:57.000000000 +0300
+++ mediawiki-1.8.2/maintenance/archives/upgradeWatchlist.php	2006-10-16 00:50:36.273804990 +0300
@@ -15,7 +15,7 @@
 # Convert watchlists to new format
 
 global $IP;
-require_once( "../LocalSettings.php" );
+require_once( "/etc/webapps/mediawiki/LocalSettings.php" );
 require_once( "$IP/Setup.php" );
 
 $wgTitle = Title::newFromText( "Rebuild links script" );
